client/src/analysis/components/DefaultFrameView.tsx

```diff
@@ -15,68 +28,144 @@ const useDefaultFrameView = ({
     const availableModes = [
         {
             text: "Average",
-            value: AnalysisTypes.SUM_FRAMES,
+            value: DefaultModes.SUM,
+        },
+        {
+            text: "SD",
+            value: DefaultModes.SD,
         },
         {
-            text: "Pick",
-            value: AnalysisTypes.PICK_FRAME,
+            text: "PICK",
```

Comment:
Please, change back to `Pick`, `PICK` is a bit too screamy for my taste :grinning: